admin/JDG_HotRod_Data_Sources.adoc

```diff
@@ -101,20 +162,79 @@ To enable the Hotrod client in the resource-adapter to communicate using SSL, co
 | TrustStorePassword | TrustStore Password
 | KeyStoreFileName   | File name of the keystore
 | KeyStorePassword   | KeyStore Password
-| SNIHostName        | [optional] SNI Host Name
+| SNIHostName        | _[optional]_ SNI Host Name
 |===
 
-=== *Using for External Materialization*
+See the link:http://infinispan.org/docs/9.0.x/infinispan_server_guide/infinispan_server_guide.html#_hot_rod_encryption[SSL Encrption] guide for more details on the properties.
 
-The following are the additional properties that need to be configured if using the Remote Cache for external materialization:
+==== *Enable using Hot Rod Authentication*
+
+This section describes how to configure the JDG resource-adapter to utilize the Hot Rod Client authentication.
+
+The JDG server will need to have its _hotrod-connector_ configured to enable accessing using authentication. 
+
+[source,xml]
+.*Example configured hotrod-connector*
+----
+<hotrod-connector socket-binding="hotrod" cache-container="default">
+   <authentication security-realm="ApplicationRealm">
+      <sasl server-name="myhotrodserver" mechanisms="DIGEST-MD5" qop="auth" />
+   </authentication>
+</hotrod-connector>
+----
+
+Note the following
+
+* server-name attribute - it is the name that the server declares to incoming clients and therefore the client configuration must match the <<AuthServerName>> property.
+* security-realm - it should be the same one defined for <<AuthApplicationRealm>>
+* mechanisms - it should be the same one defined for <<AuthSASLMechanism>>
+
+
+The following properties define how the JDG resource-adapter can be configured to talk to the JDG server based on the _hotrod_connector_.
```

Comment:
Yes, we can be more clear here that these properties will be required if using Hot Rod Authentication.